miniApp/smartNotes/models.py

```diff
@@ -1,3 +1,23 @@
 from django.db import models
 
 # Create your models here.
+class Subject(models.Model):
+    name = models.CharField(max_length=64)
+
+    def __str__(self):
+        return self.name
+
+class Mentor(models.Model):
+    name = models.CharField(max_length=64)
+
+    def __str__(self):
+        return self.name
+
+class SmartNotes(models.Model):
+    title = models.CharField(max_length=64, unique=True)
+    subjectName = models.ForeignKey(Subject, on_delete=models.CASCADE)
+    mentorName = models.ForeignKey(Mentor, on_delete=models.CASCADE)
```

Comment:
why name it `mentorName` when it's `foreignKey` attribute of Mentor model ? I would suggest you change it as `mentor`

Reply:
Ok, I've resolved it.